README.rst

```diff
@@ -113,7 +113,9 @@ The following parameters are optional:
 - :code:`rp_issue_system_url = http://bugzilla.some.com/show_bug.cgi?id={%issue_id}` - issue URL (issue_id will be filled by parameter from pytest mark)
 - :code:`rp_issue_id_marks = True` - Enables adding marks for issue ids (e.g. "issue:123456")
 - :code:`rp_verify_ssl = True` - Verify SSL when connecting to the server
-- :code:`rp_display_suite_test_file = True` In case of True, include the suite's relative file path in the launch name as a convention of "<RELATIVE_FILE_PATH>::<SUITE_NAME>". In case of False, set the launch name to be the suite name only - this flag is relevant only when "rp_hierarchy_module" flag is set to False
+- :code:`rp_display_suite_test_file = True` - In case of True, include the suite's relative file path in the launch name as a convention of "<RELATIVE_FILE_PATH>::<SUITE_NAME>". In case of False, set the launch name to be the suite name only - this flag is relevant only when "rp_hierarchy_module" flag is set to False
+- :code:`rp_mode = DEFAULT` - DEBUG or DEFAULT launch mode. DEBUG launches are displayed in a separate tab and not visible to anyone except owner
+- :code:`rp_hierarchy_dir_path_separator` - Path separator to display directories in test hierarchy. In case of empty value current system path separator will be used
```

Comment:
I think this option is excessive. We can dynamically determine system path separator in the code.

Reply:
We can, but original issue says that test names are different for MacOS and Windows. And I agree with the case that user can run tests on different systems and wants to see the same hierarchy for both systems. For those users I introduced this variable.

Reply:
The hierarchy will be the same if we apply the correct path separator to the test path parts. If you apply a non-system path separator through the rp_hierarchy_dir_path_separator option, it won't solve the problem.